internify/src/App.js

```diff
@@ -1,11 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import CreateJobPosting from './components/pages/CreateJobPosting';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-      </header>
+      <CreateJobPosting />
```

Comment:
Let's simplify this component name to `Create`

Reply:
This component will then contain all children components relating to the `Create` page